contracts/src/v0.8/dev/HeartbeatRequester.sol

```diff
@@ -20,8 +20,8 @@ interface IOffchainAggregator {
  *         is gated by permissions and this requester address needs to be whitelisted.
  */
 contract HeartbeatRequester is TypeAndVersionInterface, ConfirmedOwner {
-  event HeartbeatPermitted(address indexed permittedCaller, address indexed proxy);
-  event HeartbeatRemoved(address indexed permittedCaller);
+  event HeartbeatPermitted(address indexed permittedCaller, address indexed newProxy, address indexed oldProxy);
+  event HeartbeatRemoved(address indexed permittedCaller, address indexed removedProxy);
```

Comment:
do we need the proxy parameters indexed? I think just the permitted caller should be enough